packages/expo-localization/android/src/main/java/expo/modules/localization/LocalizationModule.kt

```diff
@@ -80,6 +69,19 @@ class LocalizationModule(context: Context) : ExportedModule(context) {
     val miuiRegion = getSystemProperty("ro.miui.region")
     return if (!TextUtils.isEmpty(miuiRegion)) {
       miuiRegion
-    } else getCountryCode(locale)
+    } else {
+      getCountryCode(locale)
+    }
+  }
+}
+
+/**
+ * Creates a shallow [Map] from the [Bundle]. Does not traverse nested arrays and bundles.
+ */
+private fun Bundle.toMap(): Map<String, Any?> {
```

Comment:
What about having a warning a method name already? 🤔 
I guess `toMap` suggests either proper deep transformation or a shallow one, and I think having more explicit method name would be more precise 😉 
```suggestion
private fun Bundle.toShallowMap(): Map<String, Any?> {
```